src/models/QueryStateModel.ts

```diff
@@ -14,7 +14,8 @@ export const QUERY_STATE_ATTRIBUTES = {
   HD: 'hd',
   HQ: 'hq',
   QUICKVIEW: 'quickview',
-  DEBUG: 'debug'
+  DEBUG: 'debug',
+  RPP: 'rpp'
```

Comment:
Not a fan of the name.

What about "numberOfResults" ? Even if it's kinda verbose, at least it seems more meaningful for end users to see this in an URL and understand what it does.